Vagrantfile

```diff
@@ -0,0 +1,15 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "fedora/32-cloud-base"
+  config.vm.provider "virtualbox" do |v|
+  	v.memory = 2048
+  	v.cpus = 2
+  end
+  config.vm.synced_folder "init_scripts/", "/init_scripts", create: false
+  config.vm.synced_folder "selotape/", "/home/vagrant/selotape", create: true
+  config.vm.provision "shell", privileged: true,
```

Comment:
Is privileged necessary ?

Reply:
Yes, this is meant to be run as root (system level configuration)

Reply:
The provisioning is already executed as root as far as I know.

Reply:
Yes, but because of the fact that one of the scripts needs root and the other doesn't, I feel like it should be explicitly stated.

Reply:
@lioramilbaum can we resolve this conversation?